model/src/main/java/com/epam/esm/model/entity/User.java

```diff
@@ -24,12 +32,22 @@
     @Column(name = "last_name", nullable = false)
     private String lastName;
 
-    @Column(table = "account", nullable = false)
+    @Column(table = "account", nullable = false, unique = true)
     private String login;
 
     @Column(table = "account", nullable = false)
     private String password;
 
+    @Column(table = "account", columnDefinition = "is_active")
+    private boolean isActive;
+
+    @ElementCollection(targetClass = Role.class, fetch = FetchType.EAGER)
+    @CollectionTable(name = "user_role",
+            joinColumns = @JoinColumn(name = "user_id"))
+    @Enumerated(EnumType.ORDINAL)
+    @Column(name = "role_id")
+    private Set<Role> roles = new HashSet<>();
```

Comment:
It would be much simplier to have only one role, but as far as I remember at our training we discussed that admin can have user rights too to have ability to use service. FAQ I studied had a  set of Roles in User. Even UserDetails interface has Set<GrantedAuthority>  field;